lib/routers/establishment/invitations.js

```diff
@@ -1,8 +1,47 @@
 const { Router } = require('express');
+const { BadRequestError } = require('@asl/service/errors');
 const permissions = require('../../middleware/permissions');
 
 const app = Router({ mergeParams: true });
 
+const validateAction = () => (req, res, next) => {
+  const allowedActions = ['cancel', 'resend'];
+  if (!allowedActions.includes(req.action)) {
+    return next(new BadRequestError());
+  }
+  next();
+};
```

Comment:
Doing the param validation in the param method per my suggestion makes the extra middleware redundant.